assets/docs/rishit-dagli/nnclr/1.md

```diff
@@ -0,0 +1,35 @@
+# Module rishit-dagli/nnclr/1
+
+NNCLR (Nearest-Neighbor Contrastive Learning of Visual Representations), a SSL approach pre-trained on STL-10.
+
+<!-- task: image-classification -->
+<!-- network-architecture: nnclr -->
+<!-- dataset: stl10 -->
+<!-- fine-tunable: false -->
+<!-- license: apache-2.0 -->
+<!-- format: saved_model_2 -->
+<!-- asset-path: https://storage.googleapis.com/rishit-dagli/nnclr_model/saved_model.tar.gz -->
+<!-- colab: https://colab.research.google.com/github/keras-team/keras-io/blob/master/examples/vision/ipynb/nnclr.ipynb -->
+
+### TF2 SavedModel
+This is a [SavedModel in TensorFlow 2 format](https://www.tensorflow.org/hub/tf2_saved_model). Using it requires TensorFlow 2 (or 1.15) and TensorFlow Hub 0.5.0 or newer.
+
+### Overview
+
+NNCLR is a an self supervised learning approach as proposed in the paper "With a Little Help from My Friends: Nearest-Neighbor Contrastive Learning of Visual Representations" [1], by Google Research and DeepMind pre-trained on STL-10.
+
+NNCLR learns self-supervised representations that go beyond single-instance positives, which allows for learning better features that are invariant to different viewpoints, deformations, and even intra-class variations. Clustering based methods offer a great approach to go beyond single instance positives, but assuming the entire cluster to be positives could hurt performance due to early over-generalization. Instead, NNCLR uses nearest neighbors in the learned representation space as positives. In addition, NNCLR increases the performance of existing contrastive learning methods like SimCLR and reduces the reliance of self-supervised methods on data augmentation strategies.
+
+### Usage
+
+This model can be used with the `hub.load` as follows.
```

Comment:
Would you mind adding what dimensions for the input and output are expected?

Reply:
Thanks for spotting this. I originally thought that people could infer the input and output types from the notebook which shows how this model was made but it makes sense to add it in the documentation.